### PR TITLE
portal/brand: Added brand portal routing forms

### PR DESCRIPTION
**Pull request by fmendozzac**
#### Type Of Change 
- [ ] Small bug fix
- [x] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [x] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [x] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [x] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

**Comment by ironArt3mis** [comment]
Rebase completed.

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/brand/src/entities/OutgoingRouting/Form.tsx [hunk 1]
```diff
@@ -18,26 +18,29 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
+    },
+    {
+      legend: _('Outgoing route'),
       fields: [
-        'type',
-        'priority',
-        'weight',
         'routingMode',
-        'prefix',
+        // 'carrier',
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/OutgoingRouting/Form.tsx:32]: @TODO or remove

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: web/portal/brand/src/entities/OutgoingRouting/Form.tsx [hunk 2]
```diff
@@ -18,26 +18,29 @@ const Form = (props: EntityFormProps): JSX.Element => {
 
   const groups: Array<FieldsetGroups | false> = [
     {
-      legend: _('Main'),
+      legend: '',
+      fields: ['company', 'routingTag'],
+    },
+    {
+      legend: _('Call destination'),
+      fields: ['type', 'routingPattern'],
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/OutgoingRouting/Form.tsx:26]: type.group must show a cuntry selector

File: web/portal/brand/src/entities/OutgoingRouting/Form.tsx [hunk 3]
```diff
@@ -18,26 +18,29 @@ const Form = (props: EntityFormProps): JSX.Element => {
...
+    },
+    {
+      legend: _('Outgoing route'),
       fields: [
-        'type',
-        'priority',
-        'weight',
         'routingMode',
```
Comment (mmadariaga) [inline web/portal/brand/src/entities/OutgoingRouting/Form.tsx:31]: Cannot be changed on edit

File: web/portal/brand/src/entities/OutgoingRouting/Form.tsx (hunk 1 above)
Comment (mmadariaga) [reply web/portal/brand/src/entities/OutgoingRouting/Form.tsx:32]: Carries is not being selected on edit

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**